### PR TITLE
Pdfjs: Add changes to use Travis CI

### PR DESCRIPTION
**Pull request by solflare**
~~This adds a continuous-integration script that will be used by cimpler to run the test suite for pdf.js. It also adds a browser_manifest.json that declares the browsers to be tested and the path to the binaries.~~

We decided to use Travis CI so the only change that's needed is to the url for the fonttools submodule.

CC @danielj41 @danielbeardsley

### CODE REVIEW

File: test/resources/browser_manifests/browser_manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "Chrome",
+    "path": "/usr/bin/google-chrome"
```
Comment (archive64) [inline test/resources/browser_manifests/browser_manifest.json]: 👍

File: test/exec/continuous-integration.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```
Comment (archive64) [inline test/exec/continuous-integration.sh]: I'm a little unfamiliar with setting up CI. What causes this script to run?

File: test/exec/continuous-integration.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+npm install
```
Comment (archive64) [inline test/exec/continuous-integration.sh]: Does this reinstall the dependencies every time?

Comment (solflare) [reply test/exec/continuous-integration.sh]: No only if they don't exists afaik.

File: test/exec/continuous-integration.sh (hunk 2 above)
Comment (solflare) [reply test/exec/continuous-integration.sh]: This script is run by cimpler which is set in the config here: https://github.com/iFixit/ifixit/pull/17556/files

File: test/exec/continuous-integration.sh (hunk 3 above)
Comment (BaseInfinity) [reply test/exec/continuous-integration.sh]: @danielj41 nope, this will be quick after the first time

File: test/exec/continuous-integration.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+# Setup the virtual frame buffer and export the DISPLAY env variable
+Xvfb :1 -screen 5 1024x768x8 &
```
Comment (BaseInfinity) [inline test/exec/continuous-integration.sh]: Schweet! Our ubuntu machines have this (selenium grid) but not sure if our CI machine does without having to install it, do we need to add something in server-templates?

**Comment by BaseInfinity** [comment]
CR 👙 looks good to me

File: test/exec/continuous-integration.sh (hunk 4 above)
Comment (danielbeardsley) [reply test/exec/continuous-integration.sh]: Please refer to our machines generically in public places. (I've edited your comment :-)

**Comment by danielbeardsley** [comment]
Hmm..  Now that I think about it...  Should we be doing Travis CI instead? Given that this is a public repo, maybe it makes more sense?

Looks like Travis is already set up for the parent fork: https://travis-ci.org/mozilla/pdf.js Which means it would be trivial for us.

**Comment by solflare** [comment]
Should I look into doing that instead then?

**Comment by solflare** [comment]
I redid this pull to have us use Travis CI which is now set up. The only change needed is to the url for the submodule.

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by davidrans** [comment]
CR ☕️

**Comment by sctice** [comment]
Wow, that ended up being easy 🎉

**Pull request merged**